sips/sip-021/sip-021-nakamoto.md

```diff
@@ -531,15 +531,18 @@ While this does induce some consolidation pressure, we believe it is the least-b
 
 # References
 
-- [1] https://eprint.iacr.org/2020/852.pdf
-- [2] https://trust-machines.github.io/wsts/wsts.pdf
+- [1] Chelsea K, Ian G (2020) FROST: Flexible Round-Optimized Schnorr Threshold Signatures. Available at https://eprint.iacr.org/2020/852.pdf and [locally][local-FROST-paper-copy] [Verified 2 February 2024]
```

Comment:
I find this a little strange to use first name and last initial for these authors.

Reply:
I just used this: https://guides.lib.monash.edu/citing-referencing/csiro-tech-rep

Reply:
Those are last name first, good catch!

Reply:
They're using "Last F", e.g. see author "Bruno Spandonide" for example.